management/server/http/api/openapi.yml

```
@@ -1806,6 +1821,38 @@ paths:
           "$ref": "#/components/responses/forbidden"
         '500':
           "$ref": "#/components/responses/internal_error"
+  /api/peers/{peerId}/accessible-peers:
+    get:
+      summary: List of accessible peers
```

```suggestion
      summary: List accessible Peers
```
This is more is sync with the title we have for the other endpoint. This will go into the left side navigation on the API docs